en/simulation/ros_interface.md

```diff
@@ -53,8 +53,9 @@ To run SITL wrapped in ROS the ROS environment needs to be updated, then launch
 
 ```sh
 cd <Firmware_clone>
-make px4_sitl_default gazebo
-source ~/catkin_ws/devel/setup.bash    // (optional)
+make px4_sitl_default
```

Comment:
Why is this step needed - ie isn't this done by the next step?

Reply:
I tried it and that's what I found

Reply:
@TSC21 Can you explain it? I'm happy to accept it, but I'd prefer to understand it first.

Reply:
@dagar?

Reply:
This is not correct. This is only correct if Firmware is built under a catkin_ws, which is not the case. So this part itself needs to be reorganized.
There are 2 ways of handling this:
1. Firmware is built under a catkin_ws using catkin build tools, which then results on the binaries amd launch files to be put under the share folder of the same workspace. Then you just need to use the src cmd as bellow so to be able to use them;
2. As this guide implies, for using this with ROS but without the Firmware included on a catkin_ws, you need to manually export the env paths. By definition, this does not make much sense, since if you are using ROS, then you probably have a catkin_ws as a developer.

Reply:
So you're saying what I found by trial and error is not correct and I'm just confused? I guess I am now :smile:

Reply:
@juli

Reply:
Just to clarify: where's this catkin_ws being created in the first place? And why we are building with make inside a catkin_ws? There's no need for this at all. You just need to build using catkin build and then source the workspace. No need to export anything else.

Reply:
Trial and error is just me following the instructions of this guide and then realizing it's not working. If I'm doing the wrong thing I'm blaming this guide.

Reply:
Ok so then the guide needs to be updated.

Reply:
You tell me how. I don't know ROS things.